cql3/restrictions/statement_restrictions.hh

```diff
@@ -46,6 +46,8 @@ private:
 
     expr::expression _new_partition_key_restrictions;
 
+    expr::single_column_restrictions_map _single_column_partition_key_restrictions;
+
```

Comment:
If this is for filtering, then eventually it should be replaced by a single expression on which we call evaluate().